File: swift/sprint0/C.swift
func getMovingAverage(_ a: [Int], _ window: Int) -> [Double] {
    // Your code
    // “ヽ(´▽｀)ノ”
}

func readInts() -> [Int] {
    return readLine()!.split(separator: " ").map { (x: Substring) -> Int in
        return Int(x)!
    }
}

func printArray<T>(_ a: [T]) {
    for element in a {
        print(element, terminator: " ")
    }
}

let size = Int(readLine()!)!
let a = readInts()
let window = Int(readLine()!)!
let averages = getMovingAverage(a, window)
printArray(averages)

File: kotlin/sprint0/C.kt
private fun movingAverage(a: List<Int>, k: Int): List<Double> {
    // Your code
    // ヽ(´▽`)/
}

fun main() {
    val size = readInt()
    val a = readInts()
    val k = readInt()
    println(movingAverage(a, k).joinToString(" "))
}

private fun readStr() = readln()
private fun readInt() = readStr().toInt()
private fun readStrings() = readStr().split(" ")
private fun readInts() = readStrings().map { it.toInt() }